### DISCUSSION TITLE
Can we create several cluster instances under the same namespace of k8s?

### DISCUSSION THREAD

**Discussion by Gnosnay** [strimzi/strimzi-kafka-operator#6783] [Q&A] [answered]
hi team, thank you so much for your time.

during the usage, i found the behavior of deployment with two Kafka clusters deploying under the same namespace is weird. 
So I wanna know is it possible for strimzi to deploy many cluster instances under the same namespace?

**Comment by scholzj** [comment, marked as answer]
You can have multiple Kafka instances in the same namespace. But only one of them should have the User and Topic operators enabled. Otherwise they might conflict with each other. The general recommendation is to not have multiple clusters in the same namespace.

**Reply by Gnosnay** [reply]
Thank you so much for your reply.  You are pretty patient and experienced. Thank you again!
